pkg/compose/start.go

```diff
@@ -69,6 +70,9 @@ func (s *composeService) start(ctx context.Context, projectName string, options
 		if err != nil {
 			return err
 		}
+		if !utils.StringContains(options.AttachTo, name) {
```

Comment:
If the attach to array is empty, that should be interpreted as starting all services right?

```suggestion
		if len(options.AttachTo) > 0 && !utils.StringContains(options.AttachTo, name) {
```

Reply:
Yes 😅 . I just noticed the red CI and realized I missed that